### ISSUE TITLE
Update readme before people waste their time and money on tuya junk

### ISSUE THREAD

**Issue by hyttysmyrkky** [andy778/EMAX-Tuya#34]
The vulnerability tuya-cloudcutter requires, was patched long ago, so it would be good to update the readme before people buy this tuya junk thinking they can easily make the device work without internet and/or in a properly configured LAN.

If this is not true, that would be valuable information too to add.

**Comment by andy778** [comment]
I agree on should not use tuya especially without and on IOT VLAN/WLAN. This project was a hardware hacking project to see if one could convert these to esphome and totally remove any tuya from the device and network,  as I in an weak moment had bought a tuya device.  ( I had only one device so I didn't try to convert with cloud-clutter after i got it to work with esplibre, as I did this conversion with soldering and reading out the binary and doing it the hard way as no one else had done it before on this exact model, I don't like to bu another of these ones to try out)

Recommendations are:
- Don't buy tuya 
- if you have a tuya trough it away or if you like hardware/software hacking you can take inspiration from this repos

Perhaps remove link to cloud clutter is the right thing to do?

**Comment by andy778** [comment]
Perhaps like (this)[https://github.com/andy778/EMAX-Tuya/tree/34-update-readme-before-people-waste-their-time-and-money-on-tuya-junk]
